File: 1. Begginer/8- Pida al usuario dos números enteros (que se guardarán en las variables n1 y n2) y muestre su producto en pantalla/main.go
package main

import "fmt"

func main() {
	var n1 int
	var n2 int
	fmt.Println("Ingrese el primer número: ")
	fmt.Scanf("%d", &n1) // function to wait until the prompt line in enteres is Scanf not Scan ¿ok?
	fmt.Println("Ingrese el segundo número: ")
	fmt.Scanf("%d", &n2)
	fmt.Println("El resultado de la multiplicación de ", n1, " y ", n2, " es: ", n1 * n2)
}

File: 1. Begginer/8- Pida al usuario dos números enteros (que se guardarán en las variables n1 y n2) y muestre su producto en pantalla/main.c
#include<stdio.h>

int main() {
	int n1;
	int n2;
	printf("%s", "Ingrese el primer número: ");
	scanf("%i", n1);
	printf("%s", "Ingrese el segundo número: ");
	scanf("%i", n2);
	printf("El producto de %i y %i es igual a %i", n1, n2, n1 * n2);
}